library/src/scala/quoted/Type.scala

```diff
@@ -5,7 +5,7 @@ import scala.reflect.ClassTag
 import scala.runtime.quoted.Unpickler.Pickled
 
 sealed abstract class Type[T] {
-  type unary_~ = T
+  type splice = T
```

Comment:
Should `splice` be publicly visible? If not, maybe rename to `$splice` or `splice$`?

Reply:
Indeed. I will change it to `$splice`.

Reply:
I will also move these outside of the visible API in a subsequent PR